src/StoryForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import request from 'superagent'
+
+class StoryForm extends Component {
+
+    constructor(){
+      super()
+      this.state = {"voteTitle": "", "voteUrl": "", "voteTotal": 0}
+      this.handleTitleChange = this.handleTitleChange.bind(this)
+      this.handleUrlChange = this.handleUrlChange.bind(this)
+      this.handleSubmit = this.handleSubmit.bind(this)
+    }
+
+    handleSubmit(e){
+      e.preventDefault()
+      this.createStory()
+    }
+
+    handleTitleChange(e){
+      this.setState({"voteTitle": e.target.value})
+    }
+
+    handleUrlChange(e){
+      this.setState({"voteUrl": e.target.value})
+    }
+
+    createStory(){
+      var that = this
+      request.post('http://master.bass-seahorse-cod.app.push.drieapp.co/stories/')
+      .set('Content-Type', 'application/json')
+      .send({name: that.state.voteTitle, url: that.state.voteUrl, size: that.state.voteTotal})
```

Comment:
yes, because we'll be referring to these things via the keys, so referring to them as one thing for the API and another thing within the app, might be confusing.

It depends if we think that there's a strong increase in readability or comprehensibility to use a different set of names within the client app.  This relates to the concept of "bounded contexts" from Eric Evan's Domain Driven Design (DDD) which indicates that it's important to be aware of the different terminologies in use in different contexts.

However the domain context of the react front end and API is arguably the same, indicating that perhaps they would be better off sharing the same terminology ...